compile warnings for nc_test/tst_utf8_phrases.c

```
gcc -DHAVE_CONFIG_H -I. -I..  -I../include -I../oc2  -I../libsrc -DTOPSRCDIR=/home/ed/tmp/netcdf-c -DTOPBINDIR= -I../liblib -I../include -I../libsrc -I/usr/local/zlib-1.2.11/include -I/usr/local/szip-2.1/include -I/usr/local/hdf5-1.10.1_memchecker/include  -g -Wall -MT tst_utf8_phrases.o -MD -MP -MF $depbase.Tpo -c -o tst_utf8_phrases.o tst_utf8_phrases.c &&\
mv -f $depbase.Tpo $depbase.Po
tst_utf8_phrases.c: In function ‘test’:
tst_utf8_phrases.c:283:9: warning: unused variable ‘i’ [-Wunused-variable]
     int i,failures = 0;
         ^
tst_utf8_phrases.c: In function ‘main’:
tst_utf8_phrases.c:333:9: warning: unused variable ‘tstcnt’ [-Wunused-variable]
     int tstcnt = 0;
         ^~~~~~
tst_utf8_phrases.c:331:12: warning: unused variable ‘status’ [-Wunused-variable]
     int i, status;
            ^~~~~~
tst_utf8_phrases.c:331:9: warning: unused variable ‘i’ [-Wunused-variable]
     int i, status;
         ^
In file included from tst_utf8_phrases.c:10:0:
At top level:
../include/err_macros.h:21:27: warning: ‘err’ defined but not used [-Wunused-variable]
 static int total_err = 0, err = 0;
                           ^~~
../include/err_macros.h:21:12: warning: ‘total_err’ defined but not used [-Wunused-variable]
 static int total_err = 0, err = 0;
            ^~~~~~~~~
```

--- a/nc_test/tst_elatefill.c
+++ b/nc_test/tst_elatefill.c
@@ -14,7 +14,6 @@
 
 #include "config.h"
 #include <nc_tests.h>
-#include "err_macros.h"
 #include <stdio.h>
 #include <netcdf.h>
 
@@ -26,7 +25,7 @@ int
 main(int argc, char **argv)
 {
     int ncid, dimid, varid, err;
-    int no_fill, fillv, buf[10];
+    int fillv;
 
     err = nc_create(FILE_NAME, NC_NETCDF4, &ncid); ERR_CHK;
     err = nc_def_dim(ncid, "dim", 10, &dimid); ERR_CHK;

--- a/nc_test/tst_utf8_phrases.c
+++ b/nc_test/tst_utf8_phrases.c
@@ -7,7 +7,6 @@
 #include <config.h>
 #include <stdlib.h>
 #include <nc_tests.h>
-#include "err_macros.h"
 #include <netcdf.h>
 #include <string.h>
 
@@ -280,7 +279,7 @@ static int
 test(const struct Test* tests, const char* title)
 {
     int status = NC_NOERR;
-    int i,failures = 0;
+    int failures = 0;
     const struct Test* p;
 
     fprintf(stderr,"Testing %s...\n",title);
@@ -328,9 +327,7 @@ fail:
 int
 main(int argc, char** argv)
 {
-    int i, status;
     int failures = 0;
-    int tstcnt = 0;
 
     printf("\n Testing UTF-8 sequences.\n");
     failures += test(utf8currency,"Currencies");

